mmv1/products/vertexai/api.yaml

```diff
@@ -277,6 +277,91 @@ objects:
                   Configuration of the snapshot analysis based monitoring pipeline running interval. The value is rolled up to full day.
 
                   A duration in seconds with up to nine fractional digits, terminated by 's'. Example: "3.5s".
+
+# Vertex AI Featurestore Entity Type Feature
+  - !ruby/object:Api::Resource
+    name: FeaturestoreEntitytypeFeature
+    base_url: 'projects/{{project}}/locations/{{region}}/featurestores/{{featurestore}}/entityTypes/{{entityType}}/features'
+    create_url: 'projects/{{project}}/locations/{{region}}/featurestores/{{featurestore}}/entityTypes/{{entityType}}/features?featureId={{name}}'
+    self_link: 'projects/{{project}}/locations/{{region}}/featurestores/{{featurestore}}/entityTypes/{{entityType}}/features/{{name}}'
```

Comment:
[MEMO] I intentionally didn't use `include_project: true` as `FeaturestoreEntitytype` does because I had the following error when running the google-beta provider unit tests. 

```
# github.com/hashicorp/terraform-provider-google-beta/google-beta [github.com/hashicorp/terraform-provider-google-beta/google-beta.test]
google-beta/resource_vertex_ai_featurestore_entitytype_feature.go:28:5: project redeclared in this block
	google-beta/resource_vertex_ai_featurestore_entitytype.go:28:5: other declaration of project
FAIL	github.com/hashicorp/terraform-provider-google-beta/google-beta [build failed]
FAIL
```

That's because the `project` variable is defined as a global package variable. Therefore, `include_project` can't be used twice currently.

https://github.com/GoogleCloudPlatform/magic-modules/blob/be60743070fed605def19c14ce1f2a047a3c233f/mmv1/templates/terraform/resource.erb#L66-L70

Reply:
I've fixed the issue in [d41b2d9](https://github.com/GoogleCloudPlatform/magic-modules/pull/6568/commits/d41b2d9dba6fb47ca3d4444d9f9d2a3eb8ecb3ec).